src/FsCodec.SystemTextJson/Serdes.fs

```diff
@@ -9,14 +9,17 @@ type Serdes private () =
     static let defaultOptions = lazy Options.Create()
     static let indentOptions = lazy Options.Create(indent = true)
 
+    /// Yields the settings used by <c>Serdes</c> when no <c>options</c> are supplied.
+    static member DefaultOptions : JsonSerializerOptions = defaultOptions.Value
+
     /// Serializes given value to a JSON string.
     static member Serialize<'T>
         (   /// Value to serialize.
             value : 'T,
             /// Use indentation when serializing JSON. Defaults to false.
             [<Optional; DefaultParameterValue null>] ?indent : bool) : string =
```

Comment:
`false` instead of `null`? (same effect though)

Reply:
Doing